tests/integration/flows/websocket-stateless.yml

```diff
@@ -0,0 +1,8 @@
+jtype: Flow
+with:
+  protocol: http 
```

Comment:
shouldn't this be `websocket`? I notice the same issue in `grpc-stateful.yml`

Reply:
Fixed. Error during copy-paste.